test/cypress/e2e/contributors.cy.js

```diff
@@ -0,0 +1,20 @@
+describe('Contributors', () => {
```

Comment:
I need a sync call to understand the value of these tests; it's not clear for me.

Reply:
Seen in sync call, thanks.